source/manual/dns.html.md

```diff
@@ -143,5 +143,41 @@ Technical 2nd Line should be notified of any planned changes via email.
   do not need to make a request to Jisc if we want to change CDN providers. Just change where
   the CNAME points to.
 
+## DNS for non-`gov.uk` domains
+
+GOV.UK also manages DNS zones for some non-`gov.uk` domains.
+
+This includes:
+
+- `independent-commission.uk`
+- `independent-inquiry.uk`
+- `public-inquiry.uk`
+- `royal-comission.uk`
+- plus various others.
+
+Some of these are not managed by Terraform. If you can't find a configuration file for the zone in [govuk-dns-config][], then you'll need to update it manually in the AWS console.
+
+1. Login to the **production** AWS console.
+
+    ```
+    $ gds aws govuk-production-poweruser -l
+    ```
+
+2. Go to [Route 53 > Hosted zones](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones) and select the zone you need to update.
```

Comment:
How do we know which zone needs updating?

Reply:
It's usually obvious from the domain, e.g. if you've been asked to delegate blah.independent-inquiry.uk you want the independent-inquiry.uk Zone.

Maybe:

```suggestion
2. Go to [Route 53 > Hosted zones](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones) and select the zone for the domain you need to update.
```

Reply:
Thanks both. I've gone with:

> 2. Go to [Route 53 > Hosted zones](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones) and select the zone for the domain you need to update.
> 
>    For example, if you've been asked to delegate `example.independent-inquiry.uk` you'll need the `independent-inquiry.uk` zone.